swarm_copy/app/database/sql_schemas.py

```diff
@@ -0,0 +1,48 @@
+"""Schemas for the chatbot."""
+
+import datetime
+import uuid
+
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String
+from sqlalchemy.orm import declarative_base, relationship
+
+Base = declarative_base()
+
+
+def uuid_to_str() -> str:
+    """Turn a uuid into a string."""
+    return uuid.uuid4().hex
+
+
+class Threads(Base):
+    """SQL table for the users thread / conversations."""
+
+    __tablename__ = "threads"
+    thread_id = Column(String, primary_key=True, default=uuid_to_str)
+    vlab_id = Column(
+        String, default="430108e9-a81d-4b13-b7b6-afca00195908"
+    )  # only default for now !
+    project_id = Column(
+        String, default="eff09ea1-be16-47f0-91b6-52a3ea3ee575"
+    )  # only default for now !
+    title = Column(String, default="New chat")
+    creation_date = Column(DateTime, default=datetime.datetime.now)
+    update_date = Column(DateTime, default=datetime.datetime.now)
```

Comment:
AFAIK the standard practice is to make sure that all timestamps stored in the DB are in UTC and only do the conversion to the local timezone when presenting it to the final user (= very likely the frontend).

We either make sure the type is timezone aware ( https://docs.sqlalchemy.org/en/20/core/type_basics.html#sqlalchemy.types.DateTime.params.timezone) or just simply always use `datetime.datetime.utcnow()` without having DB timezone awareness. Maybe the latter solution is simpler:)

Reply:
We just need to make sure not to ever use `datetime.datetime.now()` and instead use `datetime.datetime.utcnow()`

Reply:
Fixed in e60ad53

Reply:
Great:)